doc/usage.rst

```diff
@@ -124,10 +125,17 @@ options::
                         baseline_python, not changed_python.
   --append FILENAME     Add runs to an existing file, or create it if
                         it doesn't exist
+  --min-time MIN_TIME   Minimum duration in seconds of a single value, used
+                        to calibrate the number of loops
+  --same-loops SAME_LOOPS
```

Comment:
Similarly to the comment above (re. same loops), argparse positions the option here and not at the end like used to be. This because it is an option of the run command.
--manifest, -b and -p flags are option of pyperformance hence they are placed at the end.

Reply:
Dropped from this PR